Atomic/syscontainers.py

```diff
@@ -591,7 +594,7 @@ def update(self, name):
         values = info["values"]
         revision = info["revision"] if "revision" in info else None
 
-        if revision and self.args.setvalues is None:
+        if revision and self.args.setvalues is None and not self.args.force:
```

Comment:
I added a fixup patch, that move the not force condition.  And took the occasion to add a comment, for what we are doing here